cllc-public-app/ClientApp/src/app/components/sep/sep-application/event/event.component.html

```diff
@@ -45,214 +45,240 @@ <h3>Event Information</h3>
             </mat-radio-group>
         </app-field>
 
-        <mat-divider></mat-divider>
         <section formGroupName="locations">
-            <mat-card [formGroupName]="i" *ngFor="let location of locations.controls; let i = index;">
-                <mat-card-header>
-                    <fa-icon [icon]="faMapMarkerAlt" mat-card-avatar>
 
-                    </fa-icon>
-                    <mat-card-title>Location {{i + 1}}</mat-card-title>
-                </mat-card-header>
+            <mat-accordion multi>
+                <mat-expansion-panel [expanded]="true" [formGroupName]="i" class="location-panel"
+                    *ngFor="let location of locations.controls; let i = index;">
+                    <!-- <mat-expansion-panel-header> -->
+                    <h3>
+                        <fa-icon [icon]="faMapMarkerAlt" mat-card-avatar></fa-icon>
+                        Location {{i + 1}}
+                    </h3>
+                    <!-- </mat-expansion-panel-header> -->
+                    <div class="d-flex justify-content-between">
+                        <mat-card class="mat-elevation-z0 location-panel">
+                            <div class="d-flex justify-content-between">
+                                <app-field label="Event Location Name" [required]="true"
+                                    [valid]="!location.get('locationName').touched || location.get('locationName').valid"
+                                    errorMessage="Name can't be blank">
+                                    <input type="text" formControlName="locationName" class="form-control">
+                                </app-field>
+                                <div>
+                                    <fa-icon [icon]="faQuestionCircle" size="lg" (click)="tooltip.toggle()"
+                                        matTooltip="For example: community centre, sports field, wedding hall etc."
+                                        #tooltip="matTooltip">
+                                    </fa-icon>
+                                </div>
+                            </div>
+                            <app-field label="Venue Type" [required]="true"
+                                [valid]="!location.get('venueType').touched || location.get('venueType').valid"
```

Comment:
you should use `isValidOrNotTouched('venueType')` here and on the other fields to avoid repeating the validation logic. The method `isValidOrNotTouched` has been added to **FormBase** so it can be shared on all forms